config/generate-imap.sh

```diff
@@ -10,9 +10,13 @@ echo $SEP >> $FILE
 
 cat variable/01-002-realname.txt >> $FILE
 cat variable/01-003-from.txt >> $FILE
+cat variable/02-002-folder.txt >> $FILE
+
+echo "# set folder=\"imaps://domain.tld/\"" >> $FILE
```

Comment:
Wouldn't it be better to add this line to `02-002-folder.txt`?

Reply:
I think that since this line applies only to accessing mail through IMAP, it makes more sense to have the line in `generate-imap.sh`, which is IMAP-specific, rather than in `02-002-folder.txt`, which is more general.

Reply:
On the other hand, you're *reusing* the comments in `02-002-folder.txt`, which apply even in the case of an IMAP folder. Whatever you like the most.

Reply:
I do still think the line properly belongs in `generate-imap.sh` for the reason that I had previously stated. But, if you think otherwise, feel free to insert the line elsewhere.

Reply:
I guess it's @DebXWoody 's call :)

Reply:
Correct, the idea is to have those txt files general and use those files via sh-scripts in different scenarios. I don't use IMAP, I will check the Pull Request next weekend.